server.js

```diff
@@ -1,10 +1,18 @@
 /* eslint-disable no-console */
 require('dotenv').config();
 const express = require('express');
+const bodyParser = require('body-parser');
 const cors = require('cors');
 const mysql = require('mysql');
+const userRoute = require('./routes/users');
+const teamRoute = require('./routes/teams');
 
 const app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(cors());
+app.use('/users', userRoute);
+app.use('/teams', teamRoute);
```

Comment:
Do we need to import the events API calls into server.js as well?

Reply:
If the code is outside server.js, which ideally it should be then yes.